### PR TITLE
chore(github/issues): improve the display of the bug report issue template

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Improve the display of the bug report issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    id: package-concerned
+    attributes:
+      label: I am opening an issue for
+      options:
+        - Styleguide documentation
+        - Figma
+        - Icons
+        - Styles (CSS, SCSS, ...)
+        - Tokens
+        - Tools (PostCSS, Stylelint, ...)
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:22]: Change order:
- Documentation
- Design Tokens
- Tools (PostCSS, Stylelint, ...)
- Styles (CSS, SCSS, ...)
- Icon set
- Figma

Comment (caqueste) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:22]: @tiloyi Should we add another option like "Component"?

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      label: Package version
+      description: Which version of the package is affected?
+      placeholder: ex. @mozaic-ds/styles@1.41.0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: A clear and concise description of what the bug is
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:37]: ~~A clear and concise description of what the bug is~~
A clear and concise description of what the bug is.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    attributes:
+      label: Github repository
+      description: To better help you, please indicate the url of your GitHub repo
+      placeholder: ex. https://github.com/adeo/mozaic-design-system
+  - type: input
+    id: mockups
+    attributes:
+      label: Mockups
+      description: Please indicate the url to your Figma mockups to give us more context
+      placeholder: ex. https://www.figma.com/file/b09kkFmjlH6RnLEDF2vppS/%5BBA%5D-Button-%F0%9F%9F%A2?node-id=0%3A1
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:51]: ~~ex.~~
e.g.,

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        - Tokens
+        - Tools (PostCSS, Stylelint, ...)
+    validations:
+      required: true
+  - type: input
+    id: package-version
+    attributes:
+      label: Package version
+      description: Which version of the package is affected?
+      placeholder: ex. @mozaic-ds/styles@1.41.0
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:30]: ex.
e.g.,

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      label: Description
+      placeholder: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: input
+    id: github-repository
+    attributes:
+      label: Github repository
+      description: To better help you, please indicate the url of your GitHub repo
+      placeholder: ex. https://github.com/adeo/mozaic-design-system
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:45]: ex.
e.g.,

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    attributes:
+      label: Description
+      placeholder: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: input
+    id: github-repository
+    attributes:
+      label: Github repository
+      description: To better help you, please indicate the url of your GitHub repo
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:44]: ~~To better help you, please indicate the url of your GitHub repo~~
To better help you, please indicate the URL of your GitHub repo.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+      label: Mockups
+      description: Please indicate the url to your Figma mockups to give us more context
+      placeholder: ex. https://www.figma.com/file/b09kkFmjlH6RnLEDF2vppS/%5BBA%5D-Button-%F0%9F%9F%A2?node-id=0%3A1
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Tell us, what's happening?
+      placeholder: |
+        Tell us what you see
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:58]: ~~Tell us what you see~~
Tell us what you see.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What is expected?
+      description: Tell us, what did you expect to happen?
+      placeholder: Describe the expected outcome
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:66]: ~~Describe the expected outcome~~
Describe the expected outcome.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the issue
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:71]: ~~Steps to reproduce the issue~~
Steps to reproduce the issue.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  - type: textarea
+    id: code
+    attributes:
+      label: Code example
+      description: Copy here the concerned code or an example of code
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:83]: ~~Copy here the concerned code or an example of code~~
Copy here the concerned code or an example of code.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+      description: Did this behavior use to work in the previous version?
+      options:
+        - label: 'Yes'
+        - label: 'No'
+        - label: "I don't know"
+  - type: textarea
+    id: additional-comments
+    attributes:
+      label: Additional comments
+      description: Add any other comments about the problem here
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:98]: ~~Add any other comments about the problem here~~
Add any other comments about the problem here.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+      placeholder: |
+        Is this a browser specific issue? If so, please specify the browser and version.
+        Do you have specific setup for sketch, plugins, etc...
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:101]: ~~Do you have specific setup for sketch, plugins, etc...~~
Do you have a specific setup for Figma, plugins, etc.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+        Do any of these matter: operating system, IDE, package manager, HTTP server, ...?
+  - type: markdown
+    attributes:
+      value: |
+        # :raising_hand: Have you thought making a contribution?
+        **Mozaic** is an innersource project, **open to free contribution**.
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:107]: ~~innersource~~
inner source

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+        The contribution is one of the ways to get quickly a fix to your problem.
```
Comment (caqueste) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:108]: ~~The contribution is one of the ways to get quickly a fix to your problem.~~
The contribution is one of the ways to get quickly a fix to your issue.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+    id: description
+    attributes:
+      label: Description
+      placeholder: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: input
+    id: github-repository
+    attributes:
+      label: Github repository
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:43]: fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+      description: To better help you, please indicate the url of your GitHub repo
+      placeholder: ex. https://github.com/adeo/mozaic-design-system
+  - type: input
+    id: mockups
+    attributes:
+      label: Mockups
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:49]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+      description: Please indicate the url to your Figma mockups to give us more context
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:50]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 1 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:22]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 2 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:37]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 3 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:51]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 4 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:30]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 5 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:45]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 6 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:44]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 7 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:58]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 8 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:66]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 9 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:71]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 10 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:83]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 11 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:98]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 12 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:101]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 13 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:107]: Fixed

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 14 above)
Comment (tiloyi) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Fixed

**Pull request merged**